Source/CarthageKit/Project.swift

```diff
@@ -120,6 +120,9 @@ public enum ProjectEvent {
 	/// Building the project is being skipped, since the project is not sharing
 	/// any framework schemes.
 	case skippedBuilding(ProjectIdentifier, String)
+	
+	/// Building the project is being skipped because it is cached.
+	case skippedBuildingCached(ProjectIdentifier)
```

Comment:
we could do that, though it would require changing the logic a bit. currently if `--cache-builds` isn't provided we completely bypass the version file check. If we want to detect this case we'll have to always check the version file, and then spit out this case when we build anyway.

Unless you're only referring to the case where we rebuild because there was a version file mismatch, explicitly pointing out that there has been an update to account for